mrjones2014/legendary.nvim
# draft: Plugin integrations!

Resolves: #183 

## How to Test

1. Install the Neovim plugins listed in the README.md changes (links in the table lower down in the README.md)
2. Enable the plugins as shown in the changes to the config section of README.md
3. Trigger the legendary.nvim finder window
4. You should be able to find keymaps/commands/functions from the Neovim plugins you just installed, automatically, without you adding them manually

## Testing for Regressions

I have tested the following:
